paddle/framework/block.cc

```diff
@@ -0,0 +1,64 @@
+/* Copyright (c) 2016 PaddlePaddle Authors. All Rights Reserve.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License. */
+
+#include "paddle/framework/block.h"
+#include "paddle/platform/enforce.h"
+
+namespace paddle {
+namespace framework {
+
+const OpDesc* SymbolTable::NewOp() {
```

Comment:
does not really new an Op?

Reply:
New a Op Desc, SymbolTable is a container of compile-time instances, this op means an op in compile-time, that is a OpDesc.